storybook-testing/playwright.config.ts

```diff
@@ -0,0 +1,21 @@
+import { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  globalTeardown: require.resolve('./teardown'),
+  outputDir: './results',
+  testDir: '../src',
+  testMatch: /.*\.stories.test\.ts/,
+  use: {
+    baseURL: process.env.NODE_ENV === 'CI' ? 'http://10.0.0.227:6006/iframe.html?id=' : 'http://localhost:6006/iframe.html?id=',
```

Comment:
I'm just mimicking what has been done elsewhere in our tests.  I don't think this works...